iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -281,6 +295,11 @@ public void commitJobs(List<JobContext> originalContextList, Operation operation
                 .collect(Collectors.toList()));
             commitTable(table.io(), fileExecutor, output, operation);
           });
+
+      // Cleanup any merge input files.
+      for (JobContext jobContext : jobContextList) {
```

Comment:
would it make sense to move loop inside of clean method?

Reply:
Done.